core/src/main/scala/org/apache/pekko/persistence/r2dbc/internal/R2dbcExecutor.scala

```diff
@@ -58,12 +58,13 @@ import reactor.core.publisher.Mono
     }
 
   def updateBatchInTx(stmt: Statement)(implicit ec: ExecutionContext): Future[Int] = {
-    val consumer: BiConsumer[Int, Integer] = (acc, elem) => acc + elem.intValue()
+    val consumer: BiConsumer[Long, java.lang.Long] = (acc, elem) => acc + elem.intValue()
     Flux
       .from[Result](stmt.execute())
       .concatMap(_.getRowsUpdated)
-      .collect(() => 0, consumer)
+      .collect(() => 0L, consumer)
       .asFuture()
+      .map(_.intValue())
```

Comment:
raised https://github.com/apache/incubator-pekko-persistence-r2dbc/issues/90 for this

Reply:
map(_.intvalue)(ExecutionContexts.parasitic)

Reply:
done